control-plane-operator/controllers/hostedcontrolplane/kubevirt_assets/files/daemonset.yaml

```diff
@@ -0,0 +1,138 @@
+#TODO: Force DaemonSet to not run on master - see
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: kubevirt-csi-node
+  namespace: {{NAMESPACE}}
+spec:
+  selector:
+    matchLabels:
+      app: kubevirt-csi-driver
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: kubevirt-csi-driver
+    spec:
+      hostNetwork: true
+      serviceAccount: kubevirt-csi-node-sa
+      priorityClassName: system-node-critical
+      tolerations:
+        - operator: Exists
+      containers:
+        - name: csi-driver
+          securityContext:
+            privileged: true
+            allowPrivilegeEscalation: true
+          imagePullPolicy: Always
+          image: quay.io/isaacdorfman/kubevirt-csi-driver:latest
+          args:
+            - "--endpoint=unix:/csi/csi.sock"
+            - "--namespace=kubevirt-csi-driver"
+            - "--node-name=$(KUBE_NODE_NAME)"
+            - "--infra-cluster-namespace=$(INFRACLUSTER_NAMESPACE)"
+            - "--infra-cluster-kubeconfig=/var/run/secrets/infracluster/kubeconfig"
```

Comment:
you shouldn't need these at all now, just add the `--run-node-service=true --run-controller-service=false` as a replacement.